modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java

```diff
@@ -1797,7 +1797,7 @@ public void testListTemplatesWithCustomArchetypesDir() throws Exception {
 		Files.copy(
 			templateFilePath,
 			customArchetypesDirPath.resolve(
-				ProjectTemplates.TEMPLATE_BUNDLE_PREFIX + "foo.bar-1.0.4.jar"));
+				"custom.name.project.templates.foo.bar-1.0.4.jar"));
```

Comment:
can you make this not be hard-coded to 1.0.4?  that way when we update that template to 1.0.5 we don't have to fix this test.